pysiaf/tests/test_nirspec.py

```diff
@@ -40,10 +40,12 @@ def test_against_test_data():
     test_data_dir = os.path.join(TEST_DATA_ROOT, instrument)
 
 
-    include_tilt = False
+    include_tilt = True
```

Comment:
Please modify the test such that both cases are executed in a loop (code below will have to be indented of course):
`for include_tilt in [False, True]:`